# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvi/ncurses.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./PORT/linux/Makefile	1995-03-18 21:02:03.000000000 +0100
+++ ./PORT/linux/Makefile	2002-09-26 03:41:18.000000000 +0200
@@ -42,19 +42,19 @@
 # To use the curses library included in the distribution, uncomment these
 # lines.
 #
-CDEF=	-DDef_term=xDef_term -Dendwin=xendwin -Dttytype=xttytype
-CURSES=-Icurses
-OBJ12=	addbytes.o addch.o addnstr.o box.o clear.o clrtobot.o clrtoeol.o \
-	cr_put.o ctrace.o cur_hash.o curses.o delch.o deleteln.o delwin.o \
-	erase.o fullname.o getch.o getstr.o id_subwins.o idlok.o initscr.o \
-	insch.o insertln.o longname.o move.o mvwin.o newwin.o overlay.o \
-	overwrite.o putchar.o refresh.o scroll.o setterm.o standout.o \
-	toucholap.o touchwin.o tscroll.o tstp.o tty.o unctrl.o
+#CDEF=	-DDef_term=xDef_term -Dendwin=xendwin -Dttytype=xttytype
+#CURSES=-Icurses
+#OBJ12=	addbytes.o addch.o addnstr.o box.o clear.o clrtobot.o clrtoeol.o \
+#	cr_put.o ctrace.o cur_hash.o curses.o delch.o deleteln.o delwin.o \
+#	erase.o fullname.o getch.o getstr.o id_subwins.o idlok.o initscr.o \
+#	insch.o insertln.o longname.o move.o mvwin.o newwin.o overlay.o \
+#	overwrite.o putchar.o refresh.o scroll.o setterm.o standout.o \
+#	toucholap.o touchwin.o tscroll.o tstp.o tty.o unctrl.o
 
 # To use the system's curses library, or to include additional
 # routines from the system's curses library, uncomment this line.
 #
-#CULIB=	-lcurses
+CULIB=	-lncurses
 
 # To use a System V curses interface with the system's curses library,
 # uncomment this line.  Using a System V curses library is dangerous.
@@ -93,7 +93,7 @@
 
 # Some vendors use "-ltermcap", not "-ltermlib".
 bin/nvi: options.h excmd.h ${ALL}
-	${CC} ${OORG} -o $@ ${ALL} ${CULIB} -ltermcap
+	${CC} ${OORG} -o $@ ${ALL} ${CULIB}
 	(cd bin; ln -sf nvi nex)
 
 options.h:
